src/ui/public/vislib/styles/_svg.less

```diff
@@ -38,24 +38,6 @@
 
 @vis-hover-opacity: 0.8;
 
-/* SVG Element Default Styling */
-rect {
```

Comment:
i would suggest to just prefix them with `.vis-wrapper` (which is a class specific to vislib) instead of removing them

Reply:
@ppisljar are these rules actually needed? removal seems harmless

Reply:
did you check every rect or circle in our svgs :) ?

Reply:
definitely used for hover in area chart (just did a quick check)

Reply:
ok, thanks :)

I'll wrap them in the class as you suggested.